Test/DependencyCollector/FunctionalTests/FuncTest/SqlTests.cs

```diff
@@ -372,7 +387,7 @@ private void ExecuteSyncSqlTests(TestWebApplication testWebApplication, int expe
                     //// The above request would have trigged RDD module to monitor and create RDD telemetry
                     //// Listen in the fake endpoint and see if the RDDTelemtry is captured
 
-                    var allItems = DeploymentAndValidationTools.SdkEventListener.ReceiveAllItemsDuringTimeOfType<TelemetryItem<RemoteDependencyData>>(DeploymentAndValidationTools.SleepTimeForSdkToSendEvents);
+                    var allItems = DeploymentAndValidationTools.SdkEventListener.ReceiveAllItemsDuringTimeOfType<TelemetryItem<RemoteDependencyData>>(DeploymentAndValidationTools.SleepTimeForSdkToSendEvents).ToArray();
                     var sqlItems = allItems.Where(i => i.data.baseData.type == "SQL").ToArray();
```

Comment:
Array of what?  These usages of var are not clear and should have concrete types for human readability.